src/main/java/radiounit/ManagedRadioUnitRegistry.java

```diff
@@ -0,0 +1,64 @@
+package radiounit;
+
+import java.util.List;
+
+import common.AlarmStatusLevel;
+import common.RatType;
+import common.Vendor;
+
+/**
+ * Radio Unit Registry to keep track of all Radio Units
+ * 
+ * 
+ * @author esiumat
+ *
+ */
+public class ManagedRadioUnitRegistry extends AbstractManagedRadioUnitRegistry {
+
+	public ManagedRadioUnitRegistry() {
```

Comment:
This should probably be a singleton eventually.

Reply:
Possibly. Do you think there's a case where we would have multiple clients with multiple registries?

Reply:
I don't believe so - the purpose of the singleton is to ensure that there is only one list of radios.